src/tools/http_download.c

```diff
@@ -3,7 +3,7 @@
  * vim: expandtab:ts=4:sts=4:sw=4
  *
  * Copyright (C) 2012 - 2019 James Booth <boothj5@gmail.com>
- * Copyright (C) 2020 William Wennerström <william@wstrm.dev>
+ * Copyright (C) 2019 - 2023 Michael Vetter <jubalh@iodoru.org>
```

Comment:
I think Wiliam started this module probably. No license/copyright expert so not sure what to do here.